[2023-01-06 15:05:00,403] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Demo.Dag_Test scheduled__2023-01-06T15:04:00+00:00 [queued]>
[2023-01-06 15:05:00,410] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Demo.Dag_Test scheduled__2023-01-06T15:04:00+00:00 [queued]>
[2023-01-06 15:05:00,411] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-01-06 15:05:00,411] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-01-06 15:05:00,412] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-01-06 15:05:00,428] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): Dag_Test> on 2023-01-06 15:04:00+00:00
[2023-01-06 15:05:00,433] {standard_task_runner.py:52} INFO - Started process 674 to run task
[2023-01-06 15:05:00,436] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'Demo', 'Dag_Test', 'scheduled__2023-01-06T15:04:00+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/demonstration.py', '--cfg-path', '/tmp/tmppfrl5ub4', '--error-file', '/tmp/tmp__k8f3dw']
[2023-01-06 15:05:00,437] {standard_task_runner.py:80} INFO - Job 36: Subtask Dag_Test
[2023-01-06 15:05:00,486] {task_command.py:369} INFO - Running <TaskInstance: Demo.Dag_Test scheduled__2023-01-06T15:04:00+00:00 [running]> on host 4bf6c36955e2
[2023-01-06 15:05:00,546] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=Demo
AIRFLOW_CTX_TASK_ID=Dag_Test
AIRFLOW_CTX_EXECUTION_DATE=2023-01-06T15:04:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-06T15:04:00+00:00
[2023-01-06 15:05:00,548] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-01-06 15:05:00,549] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "Hello"']
[2023-01-06 15:05:00,564] {subprocess.py:85} INFO - Output:
[2023-01-06 15:05:00,566] {subprocess.py:92} INFO - Hello
[2023-01-06 15:05:00,566] {subprocess.py:96} INFO - Command exited with return code 0
[2023-01-06 15:05:00,594] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Demo, task_id=Dag_Test, execution_date=20230106T150400, start_date=20230106T150500, end_date=20230106T150500
[2023-01-06 15:05:00,609] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-01-06 15:05:00,636] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
